tests/build-workflows.py

```diff
@@ -83,7 +83,10 @@ def generate_replacements(conf, dir):
         ] = "# Omitted checking out blocks and plots repo Chia-Network/test-cache"
     if not conf["install_timelord"]:
         replacements["INSTALL_TIMELORD"] = "# Omitted installing Timelord"
-    replacements["PYTEST_PARALLEL_ARGS"] = " -n 4" if conf["parallel"] else " -n 0"
+    if conf.get("custom_parallel_n", None):
```

Comment:
Instead of having it be conditionally present we can set defaults in `testconfig.py`.

https://github.com/Chia-Network/chia-blockchain/blob/8833cc351c49a7e9340626c7c9e4715a0ecd3626/tests/testconfig.py#L1-L12

Reply:
I didn't see that file, thank you